utils/syncthing/Makefile

```diff
@@ -0,0 +1,67 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=syncthing
+PKG_VERSION:=0.14.48
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/syncthing/syncthing/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=4cd5fdc11765b87fbc08768edbb3b9b2da03d82242d613b6adb76913481f6e3f
+
+PKG_LICENSE:=MPL-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
```

Comment:
MAINTAINER line is usually above LICENSE.

Add PKG_CPE_ID:=cpe:/a:syncthing:syncthing below LICENSE_FILES.